packages/vega/package.json

```diff
@@ -14,44 +14,44 @@
   "license": "BSD-3-Clause",
   "author": "UW Interactive Data Lab (http://idl.cs.washington.edu)",
   "main": "build/vega-node.js",
-  "module": "index",
+  "module": "build/vega-core.mjs",
   "unpkg": "build/vega.min.js",
   "jsdelivr": "build/vega.min.js",
```

Comment:
Do we need a types block here or does depending on vega-typings suffice?

Reply:
I'm not the right person to answer; however, I do note that vega-util has an explicit `types` property in its `package.json` file.

Reply:
I can make sure to test it before the final release. Do you plan to have an rc release before the final release?